packages/longhorn/longhorn-1.4/generated-changes/patch/questions.yaml.patch

```diff
@@ -117,3 +117,23 @@
      description: "Specify CSI liveness probe image repository. Leave blank to autodetect."
      type: string
      label: Longhorn CSI Liveness Probe Image Repository
+@@ -179,7 +179,7 @@
+   type: string
+   default: ""
+ - variable: privateRegistry.createSecret
+-  default: "true"
++  default: "false"
+   description: "Create a new private registry secret"
+   type: boolean
+   group: "Private Registry Settings"
+@@ -824,8 +824,8 @@
+     show_if: "service.ui.type=NodePort||service.ui.type=LoadBalancer"
+     label: UI Service NodePort number
+ - variable: enablePSP
```

Comment:
Is Longhorn going to stick to using `enablePSP` rather than `global.cattle.psp.enabled` like our other charts? Considering we are bumping all charts to 102.x.x to communicate the breaking change, this would be the perfect time to change the name of this variable to be consistent across our charts. However, I'm not familiar with the implications of changing this variable and any impact it may have on Longhorn upgrades and other scenarios so please take this comment as a suggestion and not a change request.

Reply:
`enablePSP` exists in Longhorn upstream chart. 
* In that upstream chart, there is no concept Rancher `cattle` system so it doesn't less sense to rename the variable to `global.cattle.psp.enabled` in upstream chart
* The other approach is to keep the upstream the same but modify the variable name when building the rancher charts. This has the disadvantage that the upstream and rancher chart will be diverted. Additional work to verify upgrade will also needed as you mentioned.

Reply:
If the cattle concept in Longhorn is different than the cattle concept in the rest of our charts, and the variable name cannot be changed upstream because of this difference in concept, I guess it makes sense to stick to the original variable name. The benefit would be convention and better user experience since all charts would have PSP enablement done in the same way, but I would rather stick to the original name than have the Longhorn chart here deviate from uptream. Thanks for the detailed clarification!